From 9dc87096687a97738dff6e8b68e9109ad8073f92 Mon Sep 17 00:00:00 2001
From: Yong Gan <yong.gan@nxp.com>
Date: Fri, 22 Jun 2018 01:55:20 +0800
Subject: [PATCH 4056/5242] MGS-3995 [#imx-1061] Can't remove galcore when
 VIVANTE_ENABLE_DRM was enabled

commit  0c547464e51f453fefe4dbe0d21b67602b62cd4f from
https://source.codeaurora.org/external/imx/linux-imx.git

Disable VIVANTE_ENABLE_DRM if not in the android platform.

Date: Jun 21, 2018
Signed-off-by: Yong Gan yong.gan@nxp.com
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/gpu-viv/Kbuild |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/drivers/mxc/gpu-viv/Kbuild b/drivers/mxc/gpu-viv/Kbuild
index a011a78..cf795c6 100644
--- a/drivers/mxc/gpu-viv/Kbuild
+++ b/drivers/mxc/gpu-viv/Kbuild
@@ -298,7 +298,11 @@ EXTRA_CFLAGS += -DgcdSECURITY=1
 endif
 
 ifneq ($(CONFIG_DRM), )
-EXTRA_CFLAGS += -DgcdENABLE_DRM=$(VIVANTE_ENABLE_DRM)
+    ifneq ($(CONFIG_ANDROID),)
+    EXTRA_CFLAGS += -DgcdENABLE_DRM=$(VIVANTE_ENABLE_DRM)
+    else
+    EXTRA_CFLAGS += -DgcdENABLE_DRM=0
+    endif
 else
 EXTRA_CFLAGS += -DgcdENABLE_DRM=0
 endif
-- 
1.7.9.5

